stage2/01-sys-tweaks/files/iiod.service

```diff
@@ -0,0 +1,17 @@
+# iiod - Systemd init script
+#
+# Copyright (C) 2016 Analog Devices Inc.
```

Comment:
more recent year?
2020?

Reply:
I thinks i used a default file from somewhere and did not change anything to it including the copyright year so will keep this one for now.

Reply:
ack